src/main/scala/org/allenai/plugins/VersionInjectorPlugin.scala

```diff
@@ -40,6 +44,14 @@ object VersionInjectorPlugin extends AutoPlugin {
       "gitDescribe",
       "The description of the current git commit"
     )
+    val gitLocalSha1 = TaskKey[String](
+      "gitLocalSha1",
+      "Most recent commit in src"
+    )
+    val cacheKey = TaskKey[String](
+      "cacheKey",
+      "cacheKey of project"
```

Comment:
You should expand this a bit (and capitalize the first word for consistency with the other keys).